docs/guides/src/main/server/db.adoc

```diff
@@ -72,50 +75,65 @@ Database schema only accounts for Unicode strings in the following special field
 
 * Descriptions of objects
 
-Otherwise, characters are limited to those contained in database encoding which is often 8-bit. However, for some database systems, it is possible to enable UTF-8 encoding of Unicode characters and use full Unicode character set in all text fields. Often, this is counterbalanced by shorter maximum length of the strings than in case of 8-bit encodings.
+Otherwise, characters are limited to those contained in database encoding, which is often 8-bit. However, for some database systems, you can enable UTF-8 encoding of Unicode characters and use the full Unicode character set in all text fields. For a given database, this choice might result in a shorter string length than the maximum string length supported by 8-bit encodings.
+
+=== Configuring Unicode support for an Oracle database
+
+Unicode characters are supported in an Oracle database if the database was created with Unicode support in the VARCHAR and CHAR fields. For example, you configured AL32UTF8 as the database character set. In this case, the JDBC driver requires no special settings.
+
+If the database character set was not created with Unicode support, you need to configure the JDBC driver to support Unicode characters in the special fields. You configure two properties. Note that you can configure these properties as system properties or as connection properties.
 
-Some of the databases require special settings to database and/or JDBC driver to be able to handle Unicode characters. Please find the settings for your database below. Note that if a database is listed here, it can still work properly provided it handles UTF-8 encoding properly both on the level of database and JDBC driver.
+.Procedure
```

Comment:
Same as above. Please remove as it is inconsistent. If we want to use .Procedure as a heading that should be proposed as a general thing, but personally I don't really see the case for this.